Kudu.Services/Diagnostics/ProfileManager.cs

```diff
@@ -35,6 +35,16 @@ internal static class ProfileManager
 
         private static string _processName = System.Environment.ExpandEnvironmentVariables("%SystemDrive%\\msvsmon\\profiler\\VSStandardCollector.Dev14.exe");
 
+        static ProfileManager()
+        {
```

Comment:
Sorry not sure if I am understanding properly, you want me to remove this line 

 `private static readonly TimeSpan _profilingIisTimeout = TimeSpan.FromMinutes(3);`

or the static constructor ?
```
static ProfileManager()
        {
            int iisProfilingTimeout = GetIisProfilingTimeout();

            if (iisProfilingTimeout > 0)
            {
                _profilingIisTimeout = TimeSpan.FromSeconds(iisProfilingTimeout);
            }
        }
```